types/src/on_chain_config/execution_config.rs

```diff
@@ -152,7 +148,7 @@ pub enum TransactionShufflerType {
     NoShuffling,
     DeprecatedSenderAwareV1(u32),
     SenderAwareV2(u32),
-    Fairness {
+    DeprecatedFairness {
```

Comment:
why not just remove it if it's never enabled?

Reply:
Thought we enabled it briefly on testnet?  @igor-aptos  do you remember?

Reply:
yup enabled, found the perf issue, and reverted.